privacy/views/privacy_activity_view.xml

```diff
@@ -33,28 +33,34 @@
                     <group name="basic">
                         <group name="owners">
                             <field name="controller_id"/>
-                            <field
-                                name="processor_ids"
-                                widget="many2many_tags"
-                            />
-                        </group>
-                        <group name="subjects">
-                            <field name="subject_find"/>
-                            <field
-                                name="subject_domain"
-                                widget="char_domain"
-                                options='{"model": "res.partner"}'
-                                attrs='{"required": [("subject_find", "=", True)],
-                                        "invisible": [("subject_find", "=", False)]}'
-                            />
                         </group>
                     </group>
                     <notebook string="Details" name="advanced">
-                        <page string="Description">
+                        <page string="Description" name="description">
                             <group>
                                 <field name="description" nolabel="1"/>
                             </group>
                         </page>
+                        <page string="Subjects" name="subjects">
```

Comment:
They are only 2 fields. No need of extra page for that.

Reply:
@pedrobaeza for now yes but will extend with some modules. For me privacy is a base module, but will add many dependent module to have a complete activity description and management.

Reply:
The same open question to the rest of the reviewers

Reply:
I can make without page, and use separators, but think we have to organise data by type (description, subjects, processors) and have a clean small form on first with a resume of the activity.

Reply:
i would prefer a extra page

Reply:
@Axel-G what do you mean ? can you explain ?